pkg/jobs/hive/hive.go

```diff
@@ -691,37 +693,122 @@ func MonitorUpgradeStatus(client clientv1.Client, clusterName string, curator *c
 	return timeoutErr
 }
 
-func validateUpgradeVersion(client clientv1.Client, clusterName string, curator *clustercuratorv1.ClusterCurator) error {
+func validateUpgradeVersion(client clientv1.Client, clusterName string, curator *clustercuratorv1.ClusterCurator) (error, string) {
```

Comment:
```suggestion
func validateUpgradeVersion(client clientv1.Client, clusterName string, curator *clustercuratorv1.ClusterCurator) (string, error) {
```

The error should be the last return field is the convention I think? Please double check.

Reply:
Yes, you're right. Fixed.